### DISCUSSION TITLE
b5fa18cd-5fb7-4273-9b09-70379de7ef60

### DISCUSSION THREAD

**Discussion by giscus** [SSWConsulting/SSW.Rules.Content#9628] [Rules Discussions]
# b5fa18cd-5fb7-4273-9b09-70379de7ef60

In object-oriented programming (OOP), a class represents a thing. Therefore, you should use a noun for your class name as this most accurately represents what it's modelling.

https://www.ssw.com.au/rules/nouns-for-class-names/

**Comment by ricksu978** [comment]
I won't say which project it is, but we use to have a class name "DurableFunctionShaveTheYak" ...
